### PR TITLE
Switch to debian & upgrade to 2.2

### PR DESCRIPTION
**Pull request by zacblazic**
As per the title.

### CODE REVIEW

**Review by itskingori — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+
+RUN mkdir -p /var/tmp/oauth2_proxy && \
     cd /var/tmp/oauth2_proxy && \
     wget --progress=dot:mega https://github.com/bitly/oauth2_proxy/releases/download/v${OAUTH2_PROXY_VERSION}/${OAUTH2_PROXY_PKG}.tar.gz && \
     echo "${OAUTH2_PROXY_SHA} *${OAUTH2_PROXY_PKG}.tar.gz" | sha256sum -c - && \
     tar xvf ${OAUTH2_PROXY_PKG}.tar.gz && \
-    cp /var/tmp/oauth2_proxy/${OAUTH2_PROXY_PKG}/oauth2_proxy /bin/ && \
-    apk del $BUILD_PKGS && \
-    rm -rf /var/cache/apk/*
+    cp /var/tmp/oauth2_proxy/${OAUTH2_PROXY_PKG}/oauth2_proxy /bin/
```
Comment (tsu-shiuan) [inline Dockerfile]: Do we not need to cleanup `/var/tmp` here?

Comment (zacblazic) [reply Dockerfile]: Will address.

Comment (zacblazic) [reply Dockerfile]: Addressed in https://github.com/Intellection/docker-oauth2-proxy/pull/11/commits/1dddf4293e63c6aaad1465af0675f820e9df015e and https://github.com/Intellection/docker-oauth2-proxy/pull/11/commits/0f71fa5a22cec01f3580eb1aacfd501f831d0985.

**Review by itskingori — APPROVED** [review]
🏆

**Review by tsu-shiuan — APPROVED** [review]

**Pull request merged**